dev/com.ibm.ws.jca_fat_regr/fat/src/suite/r80/base/jca16/ann/AdministeredObjectMergeActionTest.java

```diff
@@ -0,0 +1,1104 @@
+                                                                                                                                                    + ".rar").addAsLibrary(traAnnEjsResourceAdapter_jar).addAsLibrary(administeredObject_SingleElement).addAsManifestResource(new File(TestSetupUtils.raDir
+                                                                                                                                                                                                                                                                                       + RarTests.TRA_jca16_ann_AdministeredObjectMergeAction_SingleElementSingleAOT
+                                                                                                                                                                                                                                                                                       + "-ra.xml"),
+                                                                                                                                                                                                                                                                              "ra.xml");
+        ShrinkHelper.exportToServer(server, "connectors", TRA_jca16_ann_AdministeredObjectMergeAction_SingleElementSingleAOT);
+
+        server.startServer("AdministeredObjectMergeActionTest.log");
+        server.waitForStringInLogUsingMark("CWWKE0002I");
+        assertNotNull("FeatureManager should report update is complete",
+                      server.waitForStringInLogUsingMark("CWWKF0008I"));
+        assertNotNull("Server should report it has started",
+                      server.waitForStringInLogUsingMark("CWWKF0011I"));
+
+        String msg = server.waitForStringInLog("J2CA7001I:.*TRA_jca16_ann_AdministeredObjectMergeAction_NoAOT.*",
+                                               server.getMatchingLogFile("messages.log"));
+        assertNotNull("Could not find resource adapter successful install message for TRA_jca16_ann_AdministeredObjectMergeAction_NoAOT.rar ",
+                      msg);
+
+        msg = server.waitForStringInLog("J2CA7001I:.*TRA_jca16_ann_AdministeredObjectMergeAction_SingleElementSingleAOT.*",
+                                        server.getMatchingLogFile("messages.log"));
+        assertNotNull("Could not find resource adapter successful install message for TRA_jca16_ann_AdministeredObjectMergeAction_SingleElementSingleAOT.rar ",
+                      msg);
+
+    }
+
+    @AfterClass
+    public static void tearDown() throws Exception {
+        if (LOGGER.isLoggable(Level.FINER)) {
+            LOGGER.entering(CLASSNAME, "tearDown");
+        }
+
+        server.stopServer("CWWKE0701E", // EXPECTED
+                          "CWWKE0700W");// EXPECTED
```

Comment:
It would be good practice here to check to make sure the server is running before calling stopServer(). 

```java
      if (server.isStarted()) {
          server.stopServer();
      }
```

Reply:
I did a quick search and added this where I found server stops